lib/werewolf/game.rb

```diff
@@ -215,12 +223,7 @@ def night_finished?
 
 
     def detect_voting_finished?
-      (living_players.size == vote_count)
-    end
-
-
-    def vote_count
-      @vote_tally.values.reduce(0) {|count, s| count + s.size}
+      (@remaining_votes.size == 0)
```

Comment:
`<=` for defensive programming? although we'd also be cool with it breaking so we could find and fix.